src/main/java/Lia.java

```diff
@@ -0,0 +1,126 @@
+import java.util.Scanner;
+
+/**
+ * Lia is a simple chatbot program that helps users manage tasks.
+ */
+public class Lia {
+
+    public static void main(String[] args) {
+        // Customizing the chatbot with the name Lia
+        String logo =
+                """
+                             ██▓     ██▓ ▄▄▄     \s
+                            ▓██▒    ▓██▒▒████▄   \s
+                            ▒██░    ▒██▒▒██  ▀█▄ \s
+                            ▒██░    ░██░░██▄▄▄▄██\s
+                            ░██████▒░██░ ▓█   ▓██▒
+                            ░ ▒░▓  ░░▓   ▒▒   ▓▒█░
+                            ░ ░ ▒  ░ ▒ ░  ▒   ▒▒ ░
+                            ░ ░    ▒ ░  ░   ▒  \s
+                                ░  ░ ░        ░  ░
+                        """;
+
+        Scanner scanner = new Scanner(System.in);
+        String input;
+        Task[] tasks = new Task[100];  // Fixed-size array to store tasks
+        int taskCount = 0;  // Counter to keep track of the number of tasks
+
+        // Greet the user with enthusiasm
+        printLine();
+        System.out.println("    Hello! I'm \n" + logo);
+        System.out.println("    What can I do for you?");
```

Comment:
Instead of leaving whitespaces use a constant String with the amount of whitespace and add them as INCIDENTION. This way UI bugs of not using enough whitespace can be avoided.